api/src/utils/get-default-value.ts

```diff
@@ -45,3 +49,19 @@ function castToBoolean(value: any): boolean {
 
 	return Boolean(value);
 }
+
+function castToObject(value: any): Record<string, any> | any[] {
+	if (typeof value === 'object') return value;
```

Comment:
We should take null/undefined and other falsy values in mind here as well. I think a simple `if (!value) return value` should do the trick